debian/control

```diff
@@ -41,7 +41,7 @@ Description: This is securedrop Qubes proxy service
 
 Package: securedrop-workstation-config
 Architecture: all
-Depends: nautilus, securedrop-keyring
+Depends: qubes-core-passwordless-root, rsyslog, mailcap, apparmor, apparmor-utils, nautilus, securedrop-keyring
```

Comment:
What is apparmor-utils needed for? I had removed it from the client in https://github.com/freedomofpress/securedrop-client/pull/1856.

Reply:
I *thought* it was needed for `aa-status` but looks like that's in the `apparmor` package.

Reply:
Yeah, https://manpages.debian.org/bookworm/apparmor-utils/index.html is the commands that it provides, I don't see any we need?